src/check_pack.c

```diff
@@ -38,16 +38,36 @@
 #endif
 
 /* Maximum size for one message in the message stream. */
-#define CK_MAX_MSG_SIZE 8192
+static int ck_max_msg_size = 0;
+#ifndef DEFAULT_MAX_MSG_SIZE
+#define DEFAULT_MAX_MSG_SIZE 4096
+#endif
 /* This is used to implement a sliding window on the receiving
  * side. When sending messages, we assure that no single message
- * is bigger than this (actually we check against CK_MAX_MSG_SIZE/2).
+ * is bigger than this.
  * The usual size for a message is less than 80 bytes.
  * All this is done instead of the previous approach to allocate (actually
  * continuously reallocate) one big chunk for the whole message stream.
  * Problems were seen in the wild with up to 4 GB reallocations.
  */
 
+void check_set_max_msg_size(int max_msg_size) {
```

Comment:
Small thing, but can you match the style of the { in this file, namely:
```
void check_set_max_msg_size(int max_msg_size()
{
...
```

Reply:
Done.